### PR TITLE
Catch known exceptions

### PR DESCRIPTION
**Pull request by jnmrclmbsjse**
This PR includes:
1. Catching of known exceptions for export and added translations for it.

Some exceptions I tried are too general to catch.
Cannot return to new
Invoiced Items

### CODE REVIEW

File: src/StoreKeeper/WooCommerce/B2C/Exports/AbstractExport.php [hunk 1]
```diff
@@ -117,17 +121,38 @@ public function run()
...
+            return new ExportException(
+                esc_html__('Failed to log in. Account data or password/hash can be wrong. You also may be banned due to multiple login errors.', I18N::DOMAIN),
+                $throwable->getCode(),
+                $throwable
+            );
+        }
+
+        if (($throwable instanceof GeneralException) && 'Permission' === $throwable->getApiExceptionClass()) {
+            return new ExportException(
+                esc_html__('No sufficient rights to call the function.', I18N::DOMAIN),
```
Comment (ikulis) [inline src/StoreKeeper/WooCommerce/B2C/Exports/AbstractExport.php:147]: You should never get this error, cos probably it means you are calling wrong (not available) function, how did you arrive to it?

File: src/StoreKeeper/WooCommerce/B2C/Exports/AbstractExport.php [hunk 2]
```diff
@@ -117,17 +121,38 @@ public function run()
...
         }
 
         return $exceptions;
     }
 
+    protected function catchKnownExceptions($throwable)
+    {
+        if ($throwable instanceof AuthException) {
+            return new ExportException(
+                esc_html__('Failed to log in. Account data or password/hash can be wrong. You also may be banned due to multiple login errors.', I18N::DOMAIN),
```
Comment (ikulis) [inline src/StoreKeeper/WooCommerce/B2C/Exports/AbstractExport.php:139]:
```suggestion
            return new PluginDisconnectedException(
                esc_html__('This channel was disconnected in StoreKeeper Backoffice, please reconnect it manually.', I18N::DOMAIN),
```

File: src/StoreKeeper/WooCommerce/B2C/Exports/AbstractExport.php (hunk 1 above)
Comment (jnmrclmbsjse) [reply src/StoreKeeper/WooCommerce/B2C/Exports/AbstractExport.php]: Removed catching of permission exception.

File: src/StoreKeeper/WooCommerce/B2C/Exports/AbstractExport.php (hunk 2 above)
Comment (jnmrclmbsjse) [reply src/StoreKeeper/WooCommerce/B2C/Exports/AbstractExport.php]: Added new exception `PluginDisconnectedException`.

**Pull request merged**